jobs/jobs-operator/jobs-operator/src/main.py

```diff
@@ -0,0 +1,66 @@
+import kopf
+from kubernetes import client as kclient
+import os
+import logging
+import sys
+
+logging.basicConfig(
+    format="%(asctime)s [%(levelname)s] %(name)s: %(message)s",
+    level=logging.INFO,
+    datefmt="%Y-%m-%d %H:%M:%S",
+    stream=sys.stdout,
+)
+logger = logging.getLogger(__name__)
+
+
+@kopf.on.create("curibio.dev", "v1", "jobrunners")  # TODO look at how to change the on create to prod.net
+def create_fn(body, spec, **kwargs):
+    # Get info from grafana object
```

Comment:
is this a hold-over comment from the demo code or is this dependent on grafana?

Reply:
that's my bad, yeah comment from demo code